src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetExceptionForHRTests.cs

```diff
@@ -17,5 +17,31 @@ public void GetExceptionForHR_EqualsErrorCode(int err)
             Exception ex = Marshal.GetExceptionForHR(err);
             Assert.Equal(err, ex.HResult);
         }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void GetExceptionForHR_ThrowExceptionForHR_ThrowsSameException()
+        {
+            int errorCode = -2146231029;
```

Comment:
Error code should _always_ be in hex. `unchecked((int)0x80131D0B)` - The reason for this is because they are very often based on Windows `HRESULT`s and they have a [specific format](https://docs.microsoft.com/en-us/windows/desktop/com/structure-of-com-error-codes) that makes understanding the error clear.

Reply:
Fixed